setup.py

```diff
@@ -222,6 +222,7 @@ def run(self):
         "jsonschema_validation": ["jsonschema"],
         "clang-format": ["clang-format"],
         "keys": ["pymdown-extensions"],
+        "libclang": ["libclang"],
```

Comment:
shouldn't this be specific to the domain (like `pyyaml` is to `[json]`)?

Reply:
Done.